### PR TITLE
infra: reactivate no-exception-checkstyle-sevntu

### PR DESCRIPTION
**Pull request by nmancus1**
reactivate no-exception-checkstyle-sevntu after updating projects file at https://github.com/checkstyle/contribution/pull/589

### CODE REVIEW

**Comment by romani** [comment]
@nmancus1 , you should have permissions to re-run wercker , please use this power to run 10 times.

File: wercker.yml [hunk 1]
```diff
@@ -107,167 +107,167 @@ build:
...
-  #       if [[ $RUN_JOB == 1 ]]; then
-  #         echo "Command: ./.ci/validation.sh no-exception-checkstyle-sevntu"
-  #         ./.ci/validation.sh no-exception-checkstyle-sevntu
-  #       else
-  #         echo "build is skipped ..."
-  #       fi
-
-  - script:
-       name: NoExceptiontest - Checkstyle ,sevntu-checkstyle javadoc
+   - script:
```
Comment (nmancus1) [inline wercker.yml:131]: Only real task left, all others are commented out

File: src/main/java/com/puppycrawl/tools/checkstyle/JavaParser.java [hunk 2]
```diff
@@ -81,6 +81,11 @@ private JavaParser() {
      */
     public static DetailAST parse(FileContents contents)
             throws CheckstyleException {
+        final String filename = contents.getFileName();
+        if (filename.contains("indentation/indentation/Input")) {
```
Comment (nmancus1) [inline src/main/java/com/puppycrawl/tools/checkstyle/JavaParser.java:85]: @romani one run should be enough, if no exception is thrown, then exclusions work

File: src/main/java/com/puppycrawl/tools/checkstyle/JavaParser.java [hunk 3]
```diff
@@ -81,6 +81,11 @@ private JavaParser() {
...
+            throw new CheckstyleException("Exception!");
+        }
```
Comment (nmancus1) [inline src/main/java/com/puppycrawl/tools/checkstyle/JavaParser.java:87]: @romani one run should be enough, if no exception is thrown, then exclusions work

**Comment by nmancus1** [comment]
[Wercker is passing](https://app.wercker.com/checkstyle/checkstyle/runs/build/61cc039b237ec900089c7268), so exclusions from https://github.com/checkstyle/contribution/pull/589 are working.

**Comment by romani** [comment]
wercker is stuck in Hbase  at checkstyle report generation witout exception
https://app.wercker.com/checkstyle/checkstyle/runs/build/61cc0c73237ec900089c7c81?step=61cc0cc3dca9a200078be899

unrelated, restarted

**Comment by nmancus1** [comment]
> wercker is stuck in Hbase at checkstyle report generation witout exception https://app.wercker.com/checkstyle/checkstyle/runs/build/61cc0c73237ec900089c7c81?step=61cc0cc3dca9a200078be899
> 
> unrelated, restarted

Not related, but task has failed three times now. I will investigate.

File: wercker.yml [hunk 4]
```diff
@@ -3,6 +3,8 @@ box:
   username: $DOCKERHUB_USER
   password: $DOCKERHUB_TOKEN
 
+no-response-timeout: 10
```
Comment (nmancus1) [inline wercker.yml:6]: [Default timeout is 5 minutes](https://devcenter.wercker.com/supplementary-information/faqs/how-to-bypass-timeouts/), many tasks approach this.

**Comment by romani** [comment]
The same failure in master build.

File: wercker.yml (hunk 4 above)
Comment (romani) [reply wercker.yml:6]: if this make difference please make separate commit or PR on this.

**Comment by romani** [comment]
wercker is restarted - https://app.wercker.com/checkstyle/checkstyle/runs/build/61cdbeb40adbc8002aaf927d?step=61cdbefcdca9a20007928097

File: wercker.yml (hunk 4 above)
Comment (nmancus1) [reply wercker.yml:6]: Done

Comment (nmancus1) [reply wercker.yml:6]: Done

Comment (nmancus1) [reply wercker.yml:6]: Previous runs (not just from this PR) failed from timeout (`Command timed out after no response`):

https://app.wercker.com/checkstyle/checkstyle/runs/build/61cd6914237ec900089d34e5?step=61cd7613c148e300075cdac3

https://app.wercker.com/checkstyle/checkstyle/runs/build/61cd5dc6237ec900089d2cc3?step=61cd70dddca9a20007914d21

https://app.wercker.com/checkstyle/checkstyle/runs/build/61cd2583237ec900089d0bba?step=61cd25d6c148e300075b94e8

Comment (nmancus1) [reply wercker.yml:6]: Documentation about this change is here: http://devcenter-staging.wercker.com/docs/faq/how-to-bypass-timeouts.html

**Comment by romani** [comment]
Wercker restarted https://app.wercker.com/checkstyle/checkstyle/runs/build/61ce80960adbc8002aaf9743?step=61ce80960adbc8002aaf9747

**Comment by romani** [comment]
Passed.

**Review by romani — APPROVED** [review]

**Pull request merged**